ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/ExecutionBuilderModule.java

```diff
@@ -170,12 +171,7 @@ public SafeFuture<HeaderWithFallbackData> builderGetHeader(
                 logReceivedBuilderBid(signedBuilderBid.getMessage());
 
                 if (isLocalPayloadValueWinning(signedBuilderBid, localPayloadValue)) {
```

Comment:
why dont we extract the builderBidValue once here and pass it around instead of the entire bid object?